bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/smartbulb/SmartBulbHandler.java

```diff
@@ -68,18 +69,18 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
         switch (channelUID.getId()) {
             case CHANNEL_POWER_SWITCH:
-                if (command instanceof OnOffType) {
-                    updateBinarySwitchState((OnOffType) command);
+                if (command instanceof OnOffType onOffCommand) {
+                    updateBinarySwitchState(onOffCommand);
                 }
                 break;
             case CHANNEL_BRIGHTNESS:
-                if (command instanceof PercentType) {
-                    updateMultiLevelSwitchState((PercentType) command);
+                if (command instanceof PercentType percentType) {
```

Comment:
For the same reason and consistency, I would do the same here:
```suggestion
                if (command instanceof PercentType percentCommand) {
```